src/database.js

```diff
@@ -270,6 +309,56 @@ Database.prototype.close = function(callback) {
   this.pool_.close().then(() => callback(leakError), callback);
 };
 
+/**
+ * @typedef {array} CreateTransactionResponse
+ * @property {BatchTransaction} 0 The {@link BatchTransaction}.
+ * @property {object} 1 The full API response.
+ */
+/**
+ * @callback CreateTransactionCallback
+ * @param {?Error} err Request error, if any.
+ * @param {BatchTransaction} transaction The {@link BatchTransaction}.
+ * @param {object} apiResponse The full API response.
+ */
+/**
+ * Create a transaction that can be used for batch querying.
+ *
+ * @param {TransactionOptions} [options] [Transaction options](https://cloud.google.com/spanner/docs/timestamp-bounds).
+ * @param {CreateTransactionCallback} [callback] Callback function.
+ * @returns {Promise<CreateTransactionResponse>}
+ *
+ * @example <caption>include:samples/batch.js</caption>
+ * region_tag:create_batch_transaction
+ */
+Database.prototype.createBatchTransaction = function(options, callback) {
```

Comment:
Is `gaxOptions` possible here?

Reply:
I'm going to go ahead and say no since we call multiple rpcs here.